docs/docs/adding-search.md

```diff
@@ -1,68 +1,70 @@
 ---
-title: Adding Search
-overview: true
+शीर्षक: खोज जोड़ना
+ओवरव्यू: true
```

Comment:
`ओवरव्यू` => `overview`

do not translate this, this will be used by to read overview from the file